packages/wrangler/e2e/pages-dev.test.ts

```diff
@@ -9,115 +9,117 @@ import { WranglerE2ETestHelper } from "./helpers/e2e-wrangler-test";
 import { fetchText } from "./helpers/fetch-text";
 import { normalizeOutput } from "./helpers/normalize";
 
-describe.each([{ cmd: "wrangler pages dev" }])("Pages $cmd", ({ cmd }) => {
-	it("should warn if no [--compatibility_date] command line arg was specified", async () => {
-		const helper = new WranglerE2ETestHelper();
-		await helper.seed({
-			"_worker.js": dedent`
+const port = await getPort();
+const inspectorPort = await getPort();
```

Comment:
The only changes in this file are to explicitly set both the port and inspector port to avoid concurrent tests from failing due to port 9229 being blocked.